tests/config/pytest_modules.yml

```diff
@@ -1,2619 +1,3577 @@
 adapterremovalfixprefix:
   - modules/nf-core/adapterremovalfixprefix/**
   - tests/modules/nf-core/adapterremovalfixprefix/**
+
```

Comment:
Don't think you need to be changing this file- probably just something funny on your branch.

Reply:
Cuz it always give me merging conflicts with the master... what should I do then?

Reply:
Probably something funny in your git history. Try literally copying in the file from master in a new commit?